<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FifteenMinTrader</class>
 <widget class="QDialog" name="FifteenMinTrader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>15分钟K线选股交易</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddStock">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加股票</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveStock">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除股票</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearStocks">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空股票列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImportStockList">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导入股票列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportStockList">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导出股票列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartTrading">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>开始交易</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblStartDate">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开始日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dteStart">
       <property name="specialValueText">
        <string/>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblEndDate">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>结束日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dteEnd">
       <property name="specialValueText">
        <string/>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="grpBuySignal">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>买点信号（需同时满足）</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="cbxBuySignalMASUp">
            <property name="text">
             <string>MAS向上</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxBuySignalMALUp">
            <property name="text">
             <string>MAL向上</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxBuySignalGoldCross">
            <property name="text">
             <string>MAS金叉MAL</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblContinuousHigherThanMA">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>连续</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spbBuySignalMAStayPeriod">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Helvetica</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblContinuousHigherThanMA_2">
            <property name="text">
             <string>根K线高于</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxBuySignalHigherThanMAS">
            <property name="text">
             <string>MAS</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxBuySignalHigherThanMAL">
            <property name="text">
             <string>MAL</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpSellSignal">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>卖点信号（需同时满足）</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="cbxSellSignalMASDown">
            <property name="text">
             <string>MAS向上</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxSellSignalMALDown">
            <property name="text">
             <string>MAL向下</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxSellSignalDeadCross">
            <property name="text">
             <string>MAS死叉MAL</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lblContinuousHigherThanMA_3">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>连续</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spbBuySignalMAStayPeriod_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Helvetica</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblContinuousHigherThanMA_4">
            <property name="text">
             <string>根K线高于</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxSellSignalHigherThanMAS">
            <property name="text">
             <string>MAS</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxSellSignalHigherThanMAL">
            <property name="text">
             <string>MAL</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QCheckBox" name="cbxClearByEarning">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>获利止盈</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spbClearEarning">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QCheckBox" name="cbxClearByLosing">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>割肉止损</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spbClearLosing">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="value">
            <number>-5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lblMAPeriodShort">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>短期均线(MAS)时长</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spbMAPeriodShort">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>65</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="lblMAPeriodLong">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>长期均线(MAL)时长</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spbMAPeriodLong">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>20</number>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>130</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblStockList">
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>90</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>股票代码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>股票名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>买点出现</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>卖点出现</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>持股涨幅</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>持股天数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>期间最高涨幅</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>期间最大跌幅</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>同期指数表现</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>跑赢指数</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
